src/components/dropdownSearch/DropdownSearchReducers.tsx

```diff
@@ -4,17 +4,15 @@ import { DropdownSearchActions, IOptionsDropdownSearchPayload } from './Dropdown
 import * as _ from 'underscore';
 import * as s from 'underscore.string';
 import { keyCode } from '../../utils/InputUtils';
-import { FixedQueue } from '../../utils/FixedQueue';
 import { multiSelectDropdownSearchReducer } from './MultiSelectDropdownSearch/MultiSelectDropdownSearchReducer';
 import { UUID } from '../../utils/UUID';
+import { deepClone } from '../../utils/CloneUtils';
 
 export interface IDropdownSearchState {
```

Comment:
are IDropdownSearchStateProps and IDropdownSearchState interfaces supposed to be identical ? if so i would suggest keeping one of the two only.